addon/serializers/web-api.js

```diff
@@ -10,10 +11,10 @@ export default DS.RESTSerializer.extend({
 
     if(isCollection) {
       payload.forEach((item) => {
-        this.extractRelationships(store, payloadWithRoot, item, primaryType);
+        this.privateExtractRelationships(store, payloadWithRoot, item, primaryType);
```

Comment:
`extractRelationships` is already a private method in the API. Oddly enough, a quick glance through the source didn't turn up any usages. I'd prefer to either:

1. Keep the method as it was named previously assuming that it wasn't breaking anything else.
2. Instead of using `privateExtractRelationships`, go with the Ember naming convention of prefixing "private" methods with an underscore. e.g. `_extractRelationships`.